app/services/l10n.js

```diff
@@ -32,7 +32,10 @@ export default class L10nService extends L10n {
       'th'      : 'ไทย',
       'vi'      : 'Tiếng Việt',
       'zh_Hans' : '中文（简体)',
-      'zh_Hant' : '中文（繁體)'
+      'zh_Hant' : '中文（繁體)',
```

Comment:
Wasn't there a discussion to use `zh_TW` and `zh_CN` etc or similar instead of `zh_Hans` which is rather uncommon?

Reply:
Yeah, but this issue is handled in another place, you can check it in my previous PR.